test/agoneum/web/controllers/session_controller_test.exs

```diff
@@ -3,7 +3,7 @@ defmodule Agoneum.Web.SessionControllerTest do
 
   alias Agoneum.Account
 
-  @create_attrs %{name: "some name", email: "some email", password: "some password"}
+  @create_attrs %{name: "some name", email: "some email", password: "some password", password_confirmation: "some password"}
```

Comment:
I think I'm going to not add that. It comes from a built-in validator from Ecto and should be tested there